Source/Server.h

```diff
@@ -728,6 +728,8 @@ class FOServer
         static void Crit_EventTurnBasedProcess( Critter* cr, Map* map, bool begin_turn );
         static void Crit_EventSmthTurnBasedProcess( Critter* cr, Critter* from_cr, Map* map, bool begin_turn );
 
+		static bool Crit_get_IsDisconnected(Critter* cr);
```

Comment:
`Cl_` prefix should be used, for consistency with other functions casting `Critter` to `Client` (`Cl_SetAccess()`, `Cl_IsKnownLoc()`, etc.)

```suggestion
		static bool Cl_get_IsDisconnected(Critter* cr);
```